docker-images/kafka/scripts/jwt.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
```

Comment:
What is the purpose of this script? Should it really be added?

Reply:
It's a little tool that allows you to peer into a JWT token. You can see some useful information - like when it expires, and who it was issued for. It's can be useful in combination with oauth.sh.

Reply:
So why should it be in the container images? Should it be in `/tools` instead?

Reply:
This script needs to connect to the authorization server using the same hostname that the Kafka Broker uses. For that reason it may need to be executed from the pod running in the same Kafka cluster.
Do we have a better candidate image?

Reply:
Running something in the broker pod is a bad practice from several different angles. The tool does not have any help, but it looks like it just base64 decodes the token. So why does it need to run from the broker pod? Why can't it run from a local machine?

What are the scenarios where one needs to run it? And who is expected to run this? In production environment only absolute minimum of people should have the access levels to do so. So it should be avoided.

Reply:
This tool can be run outside the pod, but base64 it relies on differs between platforms so having it in the image gives it a predictable environment where it's guaranteed to work. It's true that only a small portion of users will ever be interested in this tool. You'd only use it on the token you just got with `oauth.sh`, so it is only useful if `oauth.sh` is also there.
In order to connect to the Kafka Cluster using Kafka CLI tools you need Kafka image. While you maybe wouldn't use `kubectl exec` on the production broker, you'd still use the Strimzi Kafka image:

    kubectl run -ti --rm --restart=Never --image=quay.io/strimzi/kafka:latest-kafka-2.6.0 -- /bin/sh